.github/workflows/build.yml

```diff
@@ -189,6 +189,8 @@ jobs:
         - "mini_sam_m4"
         - "monster_m4sk"
         - "ndgarage_ndbit6"
+        - "nucleo_f767zi"
+        - "nucleo_h743zi_2"
```

Comment:
what is the _2 for?

Reply:
There are actually two versions of this board with different pinouts, irritatingly enough. The first has an older break-off STLink V2 and the second can't be split and has an Stlink V3. There's a number of differences besides. Micropython kinda-sorta supports the second one? But they've got a lot of mistakes either way in their implementation, it doesn't fully fit either board.

I don't have a V1 on hand, it'd be identical in layout to the F767 though.